service/sort.js

```diff
@@ -54,6 +66,30 @@ var sort =  {
 			event['score'] += timeScore;
```

Comment:
@brucepang is this timescore variable set at this point?

Reply:
Thanks for catching that. I think the code works but the naming is bad. I copied part of code from the previous part and forgot the variable name. Will make a commit soon.

Reply:
sg!